### ISSUE TITLE
Current release appears to be broken at node <10

### ISSUE THREAD

**Issue by Chuckatron** [DanielHreben/sequelize-transparent-cache#57]
I just installed the package and, upon starting, I am getting:
```
0|dev    | 2019-11-28 20:54 +00:00: /home/andrew/server/node_modules/sequelize-transparent-cache/src/cache/util.js:39
0|dev    | 2019-11-28 20:54 +00:00:       } catch { // TODO: Fix issue with JSON and BLOB columns
0|dev    | 2019-11-28 20:54 +00:00:               ^
0|dev    | 2019-11-28 20:54 +00:00: 
0|dev    | 2019-11-28 20:54 +00:00: SyntaxError: Unexpected token {
0|dev    | 2019-11-28 20:54 +00:00:     at createScript (vm.js:80:10)
0|dev    | 2019-11-28 20:54 +00:00:     at Object.runInThisContext (vm.js:139:10)
0|dev    | 2019-11-28 20:54 +00:00:     at Module._compile (module.js:617:28)
0|dev    | 2019-11-28 20:54 +00:00:     at Module._extensions..js (module.js:664:10)
0|dev    | 2019-11-28 20:54 +00:00:     at Object.require.extensions.(anonymous function) [as .js] (/usr/lib/node_modules/babel-cli/node_modules/babel-register/lib/node.js:152:7)
0|dev    | 2019-11-28 20:54 +00:00:     at Module.load (module.js:566:32)
0|dev    | 2019-11-28 20:54 +00:00:     at tryModuleLoad (module.js:506:12)
0|dev    | 2019-11-28 20:54 +00:00:     at Function.Module._load (module.js:498:3)
0|dev    | 2019-11-28 20:54 +00:00:     at Function._load (/usr/lib/node_modules/pm2/node_modules/@pm2/io/src/metrics/httpMetrics.ts:190:35)
0|dev    | 2019-11-28 20:54 +00:00:     at Module.require (module.js:597:17)
0|dev    | 2019-11-28 20:54 +00:00:     at require (internal/module.js:11:18)
0|dev    | 2019-11-28 20:54 +00:00:     at Object.<anonymous> (/home/andrew/server/node_modules/sequelize-transparent-cache/src/cache/index.js:1:44)
0|dev    | 2019-11-28 20:54 +00:00:     at Module._compile (module.js:653:30)
0|dev    | 2019-11-28 20:54 +00:00:     at Module._extensions..js (module.js:664:10)
0|dev    | 2019-11-28 20:54 +00:00:     at Object.require.extensions.(anonymous function) [as .js] (/usr/lib/node_modules/babel-cli/node_modules/babel-register/lib/node.js:152:7)
0|dev    | 2019-11-28 20:54 +00:00:     at Module.load (module.js:566:32)
0|dev    | 2019-11-28 20:54 +00:00:     at tryModuleLoad (module.js:506:12)
0|dev    | 2019-11-28 20:54 +00:00:     at Function.Module._load (module.js:498:3)
0|dev    | 2019-11-28 20:54 +00:00:     at Function._load (/usr/lib/node_modules/pm2/node_modules/@pm2/io/src/metrics/httpMetrics.ts:190:35)
0|dev    | 2019-11-28 20:54 +00:00:     at Module.require (module.js:597:17)
0|dev    | 2019-11-28 20:54 +00:00:     at require (internal/module.js:11:18)
0|dev    | 2019-11-28 20:54 +00:00:     at Object.<anonymous> (/home/andrew/server/node_modules/sequelize-transparent-cache/src/methods/class.js:1:15)
```
Typo?

**Comment by DanielHreben** [comment]
Hi! This is actually correct js syntax, supported starting at least from node 10. As you can see, all tests are passed. 
Thanks for reporting it, I'm gonna make sure older node versions is explicitly marked as not supported in package.json or actually support them, in this particular case it's very easy to go with second option but I have to check if there is other cases like this.

**Comment by Chuckatron** [comment]
Hi @DanielHreben Thanks for the clarification. It's probably about time I updated my node version, so this is a perfect time to do it!

**Comment by akbarcandra** [comment]
Hi @DanielHreben, I got the same problem here, currently, I'm working using node 8 and doesn't have the right to update the node version. Please to add support for node < 10, especially node 8, thank you 😊

**Comment by DanielHreben** [comment]
Fixed in 2.2.2. I added nodejs v8 to test grid, so hopefully, it will never happen again
